### PR TITLE
fix: file detection

### PR DESCRIPTION
**Pull request by Piotr1215**
Closes DOC-303

### CODE REVIEW

File: .github/workflows/reviewdog.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Run Misspell with Reviewdog
         uses: reviewdog/action-misspell@v1
         with:
-          github_token: ${{ secrets.GH_ACCESS_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
           reporter: github-pr-review
```
Comment (ThomasK33) [inline .github/workflows/reviewdog.yaml:39]: This will probably have to become a `github-pr-check`

Comment (Piotr1215) [reply .github/workflows/reviewdog.yaml:39]: It works also with `github-pr-review` and I think comments are a bit more readable.

Comment (ThomasK33) [reply .github/workflows/reviewdog.yaml:39]: Oh, in that case we might want to change back the vale linter to review comments

Comment (Piotr1215) [reply .github/workflows/reviewdog.yaml:39]: Good call, thank you! I forgot to add it.

**Review by ThomasK33 — APPROVED** [review]

**Pull request merged**